infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/protostream/OgmProtoStreamMarshaller.java

```diff
@@ -23,9 +23,7 @@
  */
 public final class OgmProtoStreamMarshaller extends ProtoStreamMarshaller {
 
-	//FIXME all static threadlocals should die
-	// See https://hibernate.atlassian.net/browse/OGM-1163
-	private static final ThreadLocal<SerializationContext> currentSerializationContext = new ThreadLocal<>();
+	private final ThreadLocal<SerializationContext> currentSerializationContext = new ThreadLocal<>();
```

Comment:
Why do you still need a thread local? `SerializationContext` is not concurrent safe?